### PR TITLE
date picker component

### PR DESCRIPTION
**Pull request by naremanhilles**
relates #17

### CODE REVIEW

File: Client/src/components/common/date-picker/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+function Datepicker() {
+  return (
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/date-picker/index.js:5]: you need a new line after the import statement line 2 
change the function name to  DatePicker

Comment (AbdallahAmmar96) [reply Client/src/components/common/date-picker/index.js:5]: and remember to change the function name in the export

Comment (naremanhilles) [reply Client/src/components/common/date-picker/index.js:5]: i fixed it :+1:

**Review by AbdallahAmmar96 — APPROVED** [review]

**Comment by kannanhassouna** [comment]
Where The Style Firstly,
and we need reusable component like component  takes array of items and render them

**Review by kannanhassouna — CHANGES_REQUESTED** [review]
Style

File: Client/src/components/common/date-picker/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./style.css";
+const days = ["Sat-Mon-Wedn", "Sun-Tues-Thurs"];
+function DatePicker() {
+  return (
```
Comment (AbdallahAmmar96) [inline Client/src/components/common/date-picker/index.js:5]: add new line after the import

**Review by Fatmasiam — APPROVED** [review]

File: Client/src/components/common/date-picker/index.js (hunk 2 above)
Comment (kannanhassouna) [reply Client/src/components/common/date-picker/index.js:5]: with you

File: Client/src/components/common/date-picker/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    <select className="select">
```
Comment (kannanhassouna) [inline Client/src/components/common/date-picker/index.js]: how you will get the value from the input. 
you need to an event handler, to get the value from the input. 
Article [https://reactjs.org/docs/forms.html](url) Enjoy.

File: Client/src/components/common/date-picker/index.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./style.css";
+const days = ["Sat-Mon-Wedn", "Sun-Tues-Thurs"];
```
Comment (kannanhassouna) [inline Client/src/components/common/date-picker/index.js]: remove the days from here and add them by props to the component. 
that will make your component more reusable

File: Client/src/components/common/date-picker/style.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.select {
+  font-size: 14px;
+  position: relative;
```
Comment (kannanhassouna) [inline Client/src/components/common/date-picker/style.css]: why Relative

**Review by kannanhassouna — CHANGES_REQUESTED** [review]

**Comment by kannanhassouna** [comment]
@Fatmasiam @AbdallahAmmar96  
next time review the P.R carefully

**Pull request closed without merging**